tests/disaster-recovery/sc_arbiter/test_netsplit.py

```diff
@@ -139,8 +141,15 @@ def test_netsplit(
         )
         logger.info(f"Netsplit induced at {start_time} for zones {zones}")
 
+        # wait for the ceph to be unhealthy
+        CephCluster().wait_for_ceph_health_not_ok()
+
+        # get the nodes which are present in the
+        # out of quorum zone
+        retry(CommandFailed, tries=5, delay=10)(sc_obj.get_out_of_quorum_nodes)()
+
         # note the end time (UTC)
-        if not sc_obj.check_ceph_accessibility(timeout=((duration + 5) * 60)):
+        if not sc_obj.check_ceph_accessibility(timeout=(duration * 60)):
```

Comment:
from where it will take duration and zones

Reply:
from the parameterization